lua/nvim-tree/actions/search-node.lua

```diff
@@ -0,0 +1,39 @@
+local utils = require'nvim-tree.utils'
+local git = require"nvim-tree.git"
+local renderer = require'nvim-tree.renderer'
+
+local M = {
+}
```

Comment:
```suggestion
local M = {}
```
just a little thing :smile: not really important